sdks/java/io/hadoop-file-system/src/main/java/org/apache/beam/sdk/io/hdfs/HadoopFileSystemOptions.java

```diff
@@ -87,7 +88,15 @@
         }
       }
 
+      Set<String> explodedConfDirs = Sets.newHashSet();
```

Comment:
Since this seems to be a Cloudera-centric behavior can we put a comment here explaining that and probably pointing to some reference like the file mentioned in the JIRA. Thx

Reply:
Sure, I do it right now.

Reply:
thx